### PR TITLE
mod_admin: add task queue status to admin/status

### PR DESCRIPTION
**Pull request by mworrell**
### Description

This also moves the error_count column from the task props to a database column.

Screenshot 2022-01-21 at 09 52 50

### Checklist

- [ ] documentation updated
- [ ] tests added
- [x] no BC breaks

### CODE REVIEW

File: apps/zotonic_core/src/install/z_install_update.erl [hunk 1]
```diff
@@ -734,8 +735,48 @@ rsc_language(C, Database, Schema) ->
...
+                fun({TaskId, Props}) ->
+                    case task_error_ct(z_db_pgsql:decode_value(Props)) of
+                        N when is_integer(N), N > 0 ->
+                            {ok, _} = epgsql:equery(C, "
+                                            update pivot_task_queue
+                                            set error_count = $1
+                                            where id = $2
+                                            ",
+                                            [ N, TaskId ]);
+                        _ ->
```
Comment (robvandenbogaard) [inline apps/zotonic_core/src/install/z_install_update.erl:769]: Nice to keep track of error count per task

Comment (robvandenbogaard) [inline apps/zotonic_core/src/install/z_install_update.erl:769]: Nice to keep track of error count per task

File: apps/zotonic_core/src/install/z_install_update.erl [hunk 2]
```diff
@@ -734,8 +735,48 @@ rsc_language(C, Database, Schema) ->
 
             {ok, _} = epgsql:equery(C, "
                             insert into pivot_task_queue (module, function, key)
-                            values ('z_install_update_task', 'init_language', $1)
+                            values ('z_install_update_task', 'init_language', '')
                             ",
-                            [ z_ids:id() ]),
+                            []),
```
Comment (robvandenbogaard) [inline apps/zotonic_core/src/install/z_install_update.erl:740]: Will the key be inserted at an other point?

Comment (mworrell) [reply apps/zotonic_core/src/install/z_install_update.erl:740]: There is an unique SQL index per module/function/key.
In this case there can only be a single task for this, so we can leave the key empty.

Comment (mworrell) [reply apps/zotonic_core/src/install/z_install_update.erl:740]:
```sql
    CREATE TABLE pivot_task_queue
    (
        id serial NOT NULL,
        module character varying(80) NOT NULL,
        function character varying(64) NOT NULL,
        key character varying(100) NOT NULL DEFAULT ''::character varying,
        due timestamp with time zone ,
        props bytea,
        error_count integer not null default 0,

        CONSTRAINT pivot_task_queue_pkey PRIMARY KEY (id),
        CONSTRAINT pivot_task_queue_module_function_key_key UNIQUE (module, function, key)
    )
```

**Pull request merged**